### PR TITLE
BAM-37: update with show tips feature

### PR DESCRIPTION
**Pull request by K4J24**
## **Associated JIRA tasks**

BAM-37

## **What the code does.**

Adding show tips feature while making pay request.

## **Does this code change break backwards compatibility?.**

The show tips is optional field. Only adding it to samples

### CODE REVIEW

File: versions/3_12/samples/config.py [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
+        return f'Config: address: {self.address}, store_id = {self.store_id,} api_key = {self.api_key}, terminal_id = {self.terminal_id}, show_tips = {self.show_tips}'
 
 def load_config():
     # load the configuration from the ENV variables
     config = Config(address=os.getenv('KODY_ADDRESS'),
                     store_id=os.getenv('KODY_STORE_ID'),
                     api_key=os.getenv('KODY_API_KEY'),
-                    terminal_id=os.getenv('KODY_TERMINAL_ID'))
+                    terminal_id=os.getenv('KODY_TERMINAL_ID'),
+                    show_tips=os.getenv('SHOW_TIPS'))
```
Comment (sishbi) [inline versions/3_12/samples/config.py:21]: IMO, this should be configured on a per-payment basis...

Comment (sishbi) [reply versions/3_12/samples/config.py:21]: But I guess _as an example_ this is fine

**Review by sishbi — APPROVED** [review]

File: versions/3_12/samples/config.py (hunk 1 above)
Comment (rfontes17) [reply versions/3_12/samples/config.py:21]: We could use 
`tips = input("Show tips?")`
Any input will be true or enter will be false

**Review by rfontes17 — APPROVED** [review]
Approved, with only a np to use an input to make it more dynamic if possible.

**Review by rfontes17 — APPROVED** [review]
LGTM

**Review by sishbi — APPROVED** [review]
LGTM

**Pull request merged**